addons/io_scene_gltf2/blender/imp/gltf2_blender_mesh.py

```diff
@@ -69,6 +69,17 @@ def do_primitives(gltf, mesh_idx, skin_idx, mesh, ob):
     """Put all primitive data into the mesh."""
     pymesh = gltf.data.meshes[mesh_idx]
 
+    # Use a class here, to be able to pass data by reference to hook (to be able to change them inside hook)
+    class IMPORT_mesh_options:
+        def __init__(self, vertex_colors: bool = True, skinning: bool = True, skin_into_bind_pose: bool = True, use_auto_smooth: bool = True):
+            self.vertex_colors = vertex_colors
+            self.skinning = skinning
+            self.skin_into_bind_pose = skin_into_bind_pose
+            self.use_auto_smooth = use_auto_smooth
+
+    mesh_options = IMPORT_mesh_options()
+    import_user_extensions('gather_import_mesh_options', mesh_options, gltf, pymesh, skin_idx)
+
```

Comment:
Would like some feedback on this approach

Reply:
There is already some option to export (or not export) vertex colors, skinning and normals. Maybe a new hook is not needed?
Not sure what are your use cases exactly.

Reply:
This is for importing.

Reply:
Oops, sorry :)
For normals, isn't 'shading' option what your are looking for?

Reply:
Not sure what you mean. Did I accidentally change something with normals?

Reply:
No, 
I mean that there is already a dropdown list option at import to choose how normals are taken into account. I don't know if you already see that, and why the choices are not fitting your need

Reply:
In short, the glTF files I'm trying to import already have the verts position changed based off the skin. As this importer also applies it at import time, it causes errors in the model